### PR TITLE
Support multiple devices

### PR DESCRIPTION
**Pull request by amaisano**
I own more than 1 flag device (the standard Flag, and the Bluetooth Flag). They both have the same product ids.

This just adds loops/lists to the mix, and applies the same code to every matching device. Tested with Flag and Bluetooth devices.

Note: Patterns are buggy on the bluetooth device.

### CODE REVIEW

**Comment by vmitchell85** [comment]
@amaisano first off thanks for contributing!

I was going to approve this (with minor changes requested) then I thought about it some more. Do you know if there is any way to differentiate multiple devices? I'm asking because I'm not sure all users would want all their devices set together.

**Comment by amaisano** [comment]
Hi @vmitchell85 !

Hmm. Well you could accept an argument for "index" which would be `DEVICES[i]` so that you can ask `luxafor-linux.py color -i 0 ....` or `luxafor-linux.py color -i 1 ....` to write values. Without an -i value we default to all devices?

(I know I need some cleanup here - like removing logging 😅 )

The only different between these devices are the bus address and the manufacturer string (one is "Greynut" the other is something else... "Micro" something). Otherwise _all_ property values are identical.

**Comment by vmitchell85** [comment]
@amaisano Unfortunately I only have one luxafor so I cannot test to see what all information would be available. Especially differences between the USB and bluetooth versions. Not sure I can justify purchasing a bluetooth version just for this PR 😆 .

It seems based on your code that the `usb.core.find` call would come back with an array of items if there are more than one. I think checking the length of devices, and applying it using the `i` index you suggested and even adding a `-all` flag would be good.

**Comment by amaisano** [comment]
@vmitchell85 LMK what you think - now users can choose the device for each operation if they like. So far after multiple reboots, the order of my two luxafor devices have remained the same.

**Comment by amaisano** [comment]
@vmitchell85 bumping this. Hope you are well!

**Comment by vmitchell85** [comment]
@amaisano sorry for the delay. The only thing I was still contemplating is whether we should be pulling in a library for the repeat. If it was a large codebase where it was needed in many places that would make sense, but for this little script I don't think it's worth it.

Can you remove that library and put the "double call" back in. You do that and I'll merge.

Thanks,
- Vince

File: luxafor-linux.py [hunk 1]
```diff
@@ -4,7 +4,9 @@
 import sys
 import argparse
 
-DEVICE = None
+from itertools import repeat
```
Comment (amaisano) [inline luxafor-linux.py:7]: ```suggestion```

**Review by vmitchell85 — APPROVED** [review]

**Comment by vmitchell85** [comment]
I'll try to merge tonight

**Comment by amaisano** [comment]
Thanks @vmitchell85 ! Just realized one of us should update the docs (Readme). You want to or should I?

**Comment by vmitchell85** [comment]
@amaisano Can you create another PR for it?

**Comment by amaisano** [comment]
@vmitchell85  https://github.com/vmitchell85/luxafor-python/pull/15

**Pull request merged**